backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -41,6 +41,10 @@
     private CassandraTypesProvider typesProvider;
     private Cluster cluster;
 
+    public static CassandraCluster createForManagedSingleton(CassandraModule module) {
+        return create(module, CassandraSingleton.getManagedHost(module));
+    }
+
     public static CassandraCluster create(CassandraModule module, String host, int port) {
```

Comment:
Do we still need those `create` methods to be public?

Reply:
For `public static CassandraCluster create(CassandraModule module, String host, int port)` I do get ride of that there: https://github.com/linagora/james-project/pull/1568/commits/c6101964230aba083142ef546fbc6453ebc6ffbd

For `public static CassandraCluster create(CassandraModule module, Host host)` some test suites don't really need it and can easily pass avoid using it (saving precious restarts).

Please note that I don't have clear explanations about it, and some driver dark magic / open files stuff might be the cause of it.

For sure https://github.com/linagora/james-project/pull/1568 `JAMES-2470 Use a single CassandraCluster per test class` might greatly help on this.... As always, it's awaiting reviews...